src/Microsoft.AspNetCore.SystemWebAdapters/Microsoft.AspNetCore.SystemWebAdapters.csproj

```diff
@@ -28,20 +28,33 @@
     <Compile Remove="**/*" />
     <Compile Include="Properties/PropertyInfo.cs" />
     <Compile Include="Ref.Standard.cs" />
+    <EmbeddedResource Remove="**/*" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
 
     <PackageReference Include="System.Runtime.Caching" Version="6.0.0" />
 
+    <EmbeddedResource Update="VirtualPathUtilityErrors.resx">
```

Comment:
@Tratcher I'd been told early on in the project just to inline error strings. Should we continue that or should we move things to a resource?

Reply:
I'm happy to inline them, perhaps with private constants for the ones that are shared. When I first started building this, I took across all the code from the referenced private classes and that used a lot more error messages - but once it was stripped down to just the stuff required, there weren't actually very many left.

Reply:
Yes, inline them. resx files are more trouble than they're worth at this stage.

Reply:
I've removed the resource file